Dockerfile.g3wsuite-deps.dockerfile

```diff
@@ -18,22 +18,22 @@ RUN chown root:root /tmp && chmod ugo+rwXt /tmp
 RUN apt-get update && apt install -y \
     libxml2-dev \
     libxslt-dev \
-    postgresql-server-dev-all \
     libgdal-dev \
     python3-dev \
-    libgdal30 \
+    libgdal34t64 \
     python3-gdal \
     python3-pip \
     curl \
     wait-for-it \
     gdal-bin \
     libsqlite3-mod-spatialite \
     dirmngr \
-    xvfb
+    xvfb \
+    postgresql-client
 
-# PyQGIS 3.34
+# PyQGIS
 RUN curl -sS https://download.qgis.org/downloads/qgis-archive-keyring.gpg > /etc/apt/keyrings/qgis-archive-keyring.gpg && \
-    echo "deb [signed-by=/etc/apt/keyrings/qgis-archive-keyring.gpg] https://qgis.org/ubuntu jammy main" | \
+    echo "deb [signed-by=/etc/apt/keyrings/qgis-archive-keyring.gpg] https://qgis.org/ubuntu noble main" | \
     tee /etc/apt/sources.list.d/qgis.list && \
     apt-get update && apt-get install -y python3-qgis qgis-server
```

Comment:
Maybe you could also think about specifying the exact versions to install of: `python3-qgis qgis-server`

If I pull an old g3w-suite image (today), wouldn't it always install the latest available?

Reply:
With no 'LTR' by apt repository is installed last version of QGIS and QGIS server, is not possibile set the verssion to install. I replay to your question?